lib/fog/bin/openstack.rb

```diff
@@ -20,6 +20,8 @@ def class_for(key)
         Fog::Orchestration::OpenStack
       when :baremetal
         Fog::Baremetal::OpenStack
+      when :planning
+        Fog::Openstack::Planning
```

Comment:
ah I was thinking why it can't find Fog::OpenStack::Compute

you should have there capital S in OpenStack